workos/async_client.py

```diff
@@ -17,20 +20,26 @@ class AsyncClient(BaseClient):
 
     _http_client: AsyncHTTPClient
 
-    _audit_logs: AuditLogsModule
-    _directory_sync: AsyncDirectorySync
-    _events: AsyncEvents
-    _mfa: MFAModule
-    _organizations: OrganizationsModule
-    _passwordless: PasswordlessModule
-    _portal: PortalModule
-    _sso: AsyncSSO
-    _user_management: AsyncUserManagement
-    _webhooks: WebhooksModule
```

Comment:
I thought these were necessary for mypy, but apparently not?

Reply:
Less boilerplate good.